packages/perseus/src/widgets/numeric-input/input-with-examples.tsx

```diff
@@ -0,0 +1,194 @@
+import * as PerseusLinter from "@khanacademy/perseus-linter";
+import * as React from "react";
+import {forwardRef, useImperativeHandle} from "react";
+import _ from "underscore";
+
+import {PerseusI18nContext} from "../../components/i18n-context";
+import TextInput from "../../components/text-input";
+import Tooltip, {
+    HorizontalDirection,
+    VerticalDirection,
+} from "../../components/tooltip";
+import {ClassNames as ApiClassNames} from "../../perseus-api";
+import Renderer from "../../renderer";
+import Util from "../../util";
+
+import type {LinterContextProps} from "@khanacademy/perseus-linter";
+import type {StyleType} from "@khanacademy/wonder-blocks-core";
+
+const {captureScratchpadTouchStart} = Util;
+
+type Props = {
+    value: string;
+    onChange: any;
+    className: string;
+    examples: ReadonlyArray<string>;
+    shouldShowExamples: boolean;
+    convertDotToTimes?: boolean;
+    buttonSet?: string;
+    buttonsVisible?: "always" | "never" | "focused";
+    labelText?: string;
+    onFocus: () => void;
+    onBlur: () => void;
+    disabled: boolean;
+    style?: StyleType;
+    id: string;
+    linterContext: LinterContextProps;
+};
+
+type State = {
+    focused: boolean;
+    showExamples: boolean;
+};
+
+// The InputWithExamples component is a child component of the NumericInput
+// and InputNumber components. It is responsible for rendering the UI elements
+// for the desktop versions of these widgets, and displays a tooltip with
+// examples of how to input the selected answer forms.
+const InputWithExamples = forwardRef<
+    React.RefObject<typeof InputWithExamples>,
+    Props
+>((props, ref) => {
+    // Desctructure the props to set default values
+    const {
+        shouldShowExamples = true,
+        onFocus = () => {},
+        onBlur = () => {},
+        disabled = false,
+        linterContext = PerseusLinter.linterContextDefault,
+        className = "",
+    } = props;
+
+    const context = React.useContext(PerseusI18nContext);
+    const inputRef = React.useRef<TextInput>(null);
+    const [state, setState] = React.useState<State>({
+        focused: false,
+        showExamples: false,
+    });
```

Comment:
These two variables are always set together, so I've opted to keep this pairing after the shift to the functional component. I wasn't sure of a good name to explain both of them so I went "generic", but I could easily see an argument to rename this to something like "setFocusStates".

Reply:
...Wait. If these are always set the exact same thing in multiple places ... why do we even need both? I bet we could just use "focused".

Reply:
I've updated this to just use a single `inputFocused` state.